djpsa/halo/sync.py

```diff
@@ -41,13 +41,42 @@ def _unpack_records(self, response):
         return records
 
 
+class CreateMixin:
+
+    def create(self, data, *args, **kwargs):
+        body = self._convert_fields_to_api_format(data)
```

Comment:
We've done this a lot in the past but PyCharm doesn't like it and it's right- it makes things more confusing than necessary.

PyCharm doesn't like the unresolved attribute reference:

![image](https://github.com/user-attachments/assets/b8032b1a-0459-4799-8bc5-c3ca82b8885c)

`_convert_fields_to_api_format` should go on a class that `CreateMixin` and `UpdateMixin` inherit from, and not on `HaloSynchronizer`.

Reply:
And the unresolved reference to `client` should be handled simply by adding a `client = None` to the parent class.